admin/client/App/screens/Item/components/RelatedItemsList.js

```diff
@@ -22,7 +23,9 @@ const RelatedItemsList = React.createClass({
 	getColumns () {
 		const { relationship, refList } = this.props;
 		const columns = refList.expandColumns(refList.defaultColumns);
-		return columns.filter(i => i.path !== relationship.refPath);
+		const refListNameColumn = columns.filter(i => i.path === refList.nameField.path);
+		const refListRelationshipFieldColumn = columns.filter(i => i.path === relationship.refPath);
```

Comment:
For clarity, I'd prefer those two to be:

```JS
columns.filter((column) => column.something === somethingElse);
```

(am not too fussed about the parentheses, but why call the variable inside the filter `i` when it's a `column`?)